infrastructure/docker/traffic_ops/run.sh

```diff
@@ -36,216 +38,255 @@
+    },
+    {
+      "LDAP Admin Password":"",
+      "config_var":"admin_pass",
+      "hidden":"1"
+    },
+    {
+      "LDAP Search Base":"",
+      "config_var":"search_base"
+    }
+  ],
+  "/opt/traffic_ops/install/data/json/users.json":[
+    {
+      "Administration username for Traffic Ops":"$ADMIN_USER",
+      "config_var":"tmAdminUser"
+    },
+    {
+      "Password for the admin user":"$ADMIN_PASS",
+      "config_var":"tmAdminPw",
+      "hidden":"1"
+    }
+  ],
+  "/opt/traffic_ops/install/data/profiles/":[
+    {
+      "Add custom profiles?":"no",
+      "config_var":"custom_profiles"
+    }
+  ],
+  "/opt/traffic_ops/install/data/json/openssl_configuration.json":[
+    {
+      "Do you want to generate a certificate?":"yes",
+      "config_var":"genCert"
+    },
+    {
+      "Country Name (2 letter code)":"$CERT_COUNTRY",
+      "config_var":"country"
+    },
+    {
+      "State or Province Name (full name)":"$CERT_STATE",
+      "config_var":"state"
+    },
+    {
+      "Locality Name (eg, city)":"$CERT_CITY",
+      "config_var":"locality"
+    },
+    {
+      "Organization Name (eg, company)":"$CERT_COMPANY",
+      "config_var":"company"
+    },
+    {
+      "Organizational Unit Name (eg, section)":"",
+      "config_var":"org_unit"
+    },
+    {
+      "Common Name (eg, your name or your server's hostname)":"$HOSTNAME",
+      "config_var":"common_name"
+    },
+    {
+      "RSA Passphrase":"passphrase",
+      "config_var":"rsaPassword",
+      "hidden":"1"
+    }
+  ],
+  "/opt/traffic_ops/install/data/json/profiles.json":[
+    {
+      "Traffic Ops url":"https://$HOSTNAME",
+      "config_var":"tm.url"
+    },
+    {
+      "Human-readable CDN Name.  (No whitespace, please)":"cdn",
+      "config_var":"cdn_name"
+    },
+    {
+      "Health Polling Interval (milliseconds)":"8000",
+      "config_var":"health_polling_int"
+    },
+    {
+      "DNS sub-domain for which your CDN is authoritative":"$HOSTNAME.$DOMAIN",
+      "config_var":"dns_subdomain"
+    },
+    {
+      "TLD SOA admin":"traffic_ops",
+      "config_var":"soa_admin"
+    },
+    {
+      "TrafficServer Drive Prefix":"/dev/ram",
+      "config_var":"driver_prefix"
+    },
+    {
+      "TrafficServer RAM Drive Prefix":"/dev/ram",
+      "config_var":"ram_drive_prefix"
+    },
+    {
+      "TrafficServer RAM Drive Letters (comma separated)":"1",
+      "config_var":"ram_drive_letters"
+    },
+    {
+      "Health Threshold Load Average":"25",
+      "config_var":"health_thresh_load_avg"
+    },
+    {
+      "Health Threshold Available Bandwidth in Kbps":"1750000",
+      "config_var":"health_thresh_kbps"
+    },
+    {
+      "Traffic Server Health Connection Timeout (milliseconds)":"2000",
+      "config_var":"health_connect_timeout"
+    }
+  ]
+}
+	ENDOFMESSAGE
+
+	# TODO determine if term, user are necessary
+	export TERM=xterm && export USER=root && /opt/traffic_ops/install/bin/postinstall -cfile "$postinstall_input_file"
```

Comment:
yes -- I've wanted to do something about that in postinstall -- shouldn't need $USER -- can use perl's var for UID (I think it's `$<`?)